modules/search/client/views/search-input.client.view.html

```diff
@@ -1,6 +1,18 @@
 <div class="form-group search-form-group search-place"
      ng-class="{ 'has-warning': search.locationNotFound }">
   <div class="input-group input-location">
+    <span class="input-group-btn"
+      style="border-left-width: 1px; border-left-style: solid; border-color: #12b591;">
+      <a type="button"
+        href="/search/users"
+        class="btn btn-lg btn-default"
+        tooltip-enable="::search.screenWidth >= 768"
+        tooltip-placement="bottom"
+        uib-tooltip="Go to search users"
+        aria-label="Go to search users by username">
+        <i class="icon-users"></i>
+      </a>
+    </span>
```

Comment:
I feel like we could take a little different route with discoverability/linking map search and user search to each other.

While these function well I think we can improve them to work around these drawbacks:

- It's not clear what happens from the button or that it even _is_ a button:

<img width="389" alt="screenshot 2018-12-30 at 20 41 14" src="https://user-images.githubusercontent.com/87168/50550238-3fa51080-0c75-11e9-8822-2a6ded80bb1a.png">
<img width="361" alt="screenshot 2018-12-30 at 20 41 24" src="https://user-images.githubusercontent.com/87168/50550239-3fa51080-0c75-11e9-99b4-40eaa89dd0ce.png">

- It looks like it's part of the search input and there to provide visual symbol for thes earch rather than it's a way to access completely different major interface.

The tooltip helps little but we cannot rely on that; especially on mobile where we don't have hover-state. Seems like it has issues also on the desktop:

<img width="364" alt="screenshot 2018-12-30 at 20 41 32" src="https://user-images.githubusercontent.com/87168/50550241-53507700-0c75-11e9-884c-7cce941771b5.png">

We can assume most folks not seeing these tooltips so we need to build UI that's clear without them and hide only additional, optional information in tooltips.

The intent at the user search side is little bit more clearer but I'd say it has similar issues and it makes the search input rather busy with all the icons:

![image](https://user-images.githubusercontent.com/87168/50550406-94965600-0c78-11e9-812b-70f32b718dca.png)

I would suggest to hide these UIs for now and e.g. add link to this feature only behind user-dropdown. That's not very discoverable of course so we'll need to work more on it, but we can have different PR for that so that we don't slow down merging this.

I know you're AFK right now; hope you don't mind if I go ahead and do those PRs and we can still add these inputs in some format if we so conclude! :-)

Reply:
Work in progress here: https://github.com/Trustroots/trustroots/pull/1020